docker/build/Dockerfile-data-dev

```diff
@@ -59,7 +59,8 @@ ENV softwareVersion=SOFTWARE-VERSION-NOT-SET \
     simdatadir_external=/path/to/external/simdata/ \
     simdataCacheSize="simdataCacheSize-not-set" \
     webDataPort="webDataPort-not-set" \
-    servertype="servertype-not-set"
+    servertype="servertype-not-set" \
+    s3ProxyPortExternal="80"
```

Comment:
When writing the export URL for accessing the N5 sim data, the port is required. May want to change the default port though from 80 to something else. Or set in some .env file.

Reply:
It seems that localconfig_realslurm_oracle.sh creates an .env file that can be used in production. If that is the case then a default of "80" isn't that bad cause it can easily just be changed in the .env file.